app/src/main/res/values/dimens.xml

```diff
@@ -3,15 +3,22 @@
     <dimen name="huge_margin">32dp</dimen>
     <dimen name="large_margin">24dp</dimen>
     <dimen name="medium_margin">16dp</dimen>
+    <dimen name="smedium_margin">12dp</dimen>
```

Comment:
`smedium` literówka?

Reply:
Zamierzone. Jest pomiędzy small i medium, łatwiej znaleźć i się domyślić z taką nazwą.

Reply:
To może dać numery zamiast nazw?

Reply:
Nazwy klasy powinny być semantyczne. Zmiana rozmiaru nie powinna wymuszać zmiany nazwy klasy. Są juz przetarte scieżki - klasę ```smedium``` zmień na ```small``` a ```small``` na ```x_small```